easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2020b-Perl-5.32.0.eb

```diff
@@ -0,0 +1,72 @@
+##
+# This file is an EasyBuild recipy as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# https://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# Updated to use gompi-2020b toolchain:
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'HH-suite'
+version = '3.3.0'
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
No need for the `versionsuffix` imho (we only use a single Perl version per generation of easyconfigs), but you'll also need to remove the old `HH-suite-3.3.0-gompi-2020b.eb` (first)

Reply:
Ok. I will remove that.